docs/reference/commandline/cli.md

```diff
@@ -160,6 +160,14 @@ property is not set, the client falls back to the default table
 format. For a list of supported formatting directives, see
 [**Formatting** section in the `docker stats` documentation](stats.md)
 
+The property `secretFormat` specifies the default format for `docker
+secret ls` output. When the `--format` flag is not provided with the
+`docker secret ls` command, Docker's client uses this property. If this
+property is not set, the client falls back to the default table
+format. For a list of supported formatting directives, see
+[**Formatting** section in the `docker secret ls` documentation](secret_ls.md)
+
```

Comment:
Can you add an example as well in the sample JSON below? (https://github.com/docker/docker/pull/31552/files#diff-578706828b816c40dd41b66896ab92c5R214)